### PR TITLE
Polyfill Buffer for Browser Compatibility 

### PR DESCRIPTION
**Pull request by natenichols**
## High Level Overview of Change
Move from NodeJS Buffer to a browser compatable [Buffer](https://www.npmjs.com/package/buffer)

### Context of Change
Instead of relying on webpacking to make libraries browser compatible, this PR uses a buffer library that will polyfill when being used in the browser.

### Type of Change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Test Plan
Add browser compatibility tests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "big-integer": "^1.6.48",
     "create-hash": "^1.2.0",
     "decimal.js": "^10.2.0",
+    "buffer": "5.6.0",
```
Comment (intelliot) [inline package.json:18]: why 5.6.0?

File: src/binary.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { sha512Half, transactionID } from "./hashes";
 import { FieldInstance } from "./enums";
 import { STObject } from "./types/st-object";
 import { JsonObject } from "./types/serialized-type";
+import { Buffer } from "buffer/";
```
Comment (intelliot) [inline src/binary.ts:12]: why the trailing `/`?

File: src/types/account-id.ts [hunk 3]
```diff
@@ -62,7 +63,9 @@ class AccountID extends Hash160 {
    * @returns the base58 string defined by this.bytes
    */
   toBase58(): string {
-    return encodeAccountID(this.bytes);
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    return encodeAccountID(this.bytes as any);
```
Comment (intelliot) [inline src/types/account-id.ts:67]: seems like we could make an improvement to `ripple-address-codec`?

File: package.json (hunk 1 above)
Comment (natenichols) [reply package.json:18]: Updating to `6.0.2`, which is latest

File: src/binary.ts (hunk 2 above)
Comment (natenichols) [reply src/binary.ts:12]: see [usage](https://www.npmjs.com/package/buffer)

File: src/types/account-id.ts (hunk 3 above)
Comment (natenichols) [reply src/types/account-id.ts:67]: Lets discuss this one in person.

**Review by intelliot — APPROVED** [review]

**Pull request merged**